about.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/bootstrap.min.css">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>About Us & Our Team</title>
+</head>
+
+<body>
+    <main class="container-fluid bg-gray row mx-0">
+        <section class="align-items-center justify-content-center">
+            <h1>ABOUT US</h1>
+            <p class="bg-white py-3 px-3 border-gray my-3">Our name was chosen to convey the restaurant’s welcoming
+                environment,
+                as it truly
+                is
+                «the
+                address» for
+                friends, colleagues and tourists to meet up at any time of day. At BlaBla we serve innovative breakfast,
+                brunch, lunch and dinner
+                menus, as well as pâtisserie created on site in the classic French tradition. Our diverse beverage
+                program
+                includes artisanal cocktails, fruit and vegetable blends, spritzers and one-of- a-kind coffee drinks
+                prepared by an award-winning barista team.</p>
+            <p>Please contact us by Email for any questions and suggestions.<br><span
+                    class="fw-bold">blabla@restaurant.com</span></p>
+        </section>
+        <section class=" container gap-4" id="team">
+            <a class="h2-wrapper" href="about.html">
+                <h2>Our Team</h2>
+                <div class="line"></div>
+            </a>
+        </section>
+    </main>
+    <footer class="copyright py-4 text-start d-flex justify-content-center">
+        <p class="col-8">2021,<br>1065 Ave of the Americas, New York, NY 10018,<br>BlaBla Restaurant</p>
+    </footer>
```

Comment:
- [ ] The navigation bar and the footer is not displaying on your pages, please kindly fix the issue.

Reply:
Hi, this and almost all your other comments are not related to today's task. I think you make evaluation based on wrong criteria.